modules/apps/analytics/analytics-batch-export-import-test/src/testIntegration/java/com/liferay/analytics/batch/exportimport/internal/engine/test/ExpandoColumnAnalyticsDXPEntityBatchEngineTaskItemDelegateTest.java

```diff
@@ -121,19 +111,16 @@ public void testSendExpandoColumnAnalytics() throws Exception {
 						).put(
 							"syncedUserFieldNames",
 							new String[] {
-								"customField", "testField", "customField5"
+								"customField", "customField5", "testField"
 							}
 						).build())) {
 
-			Pagination pagination = Pagination.of(1, 7);
-
 			Page<DXPEntity> resultPage = _batchEngineTaskItemDelegate.read(
-				filter, pagination, sorts, parameter, search);
+				null, Pagination.of(1, 7), null, Collections.emptyMap(), null);
+
+			Page<DXPEntity> dxpEntitiesPage = resultPage.getItens();
```

Comment:
typo on getItems. Please check the return type of getItems. i don't think it returns Page<DXPEntity>